OutOfSchool/OutOfSchool.IdentityServer/db-institutionshierarchyscript.txt

```diff
@@ -0,0 +1,1034 @@
+﻿USE out_of_school;
```

Comment:
rename the extension to `.sql` and maybe move to `/Config` folder in repository root as it's not part of the identity server.

Reply:
Ok ;)